src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,11 +1,22 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+
     private String title;
+
+    @OneToMany
```

Comment:
The `@OneToMany` annotation should specify the `mappedBy` attribute to indicate the field that owns the relationship in the `Actor` entity. Additionally, consider using `fetch = FetchType.LAZY` to optimize performance by loading actors only when needed.